schema/v1.1.0/dataset_config_models_extended.py

```diff
@@ -0,0 +1,353 @@
+from __future__ import annotations
+
+import asyncio
+import re
+from typing import Dict, List, Optional, Set, Tuple
+
+import aiohttp
+import requests
+from dataset_config_models import (
+    Annotation,
+    AnnotationConfidence,
+    AnnotationEntity,
+    AnnotationObject,
+    AnnotationSource,
+    Author,
+    Container,
+)
+from pydantic import field_validator, model_validator
+from typing_extensions import Self
+
+CELLULAR_COMPONENT_GO_ID = "GO:0005575"
+
+# Network requests: stores whether or not the IDs are valid
+running_orcid_list: Dict[str, bool] = {}
+running_go_id_list: Dict[str, bool] = {}
+running_publication_list: Dict[str, bool] = {}
+
+running_network_validation = False
+
+# ==============================================================================
+# Annotation Object Validation
+# ==============================================================================
+
+
+def validate_go_id(go_id: str) -> bool:
```

Comment:
We could use the `@cache` here instead of maintaining a running list. 

ref: https://docs.python.org/3/library/functools.html

Reply:
Done!